.travis.yml

```diff
@@ -3,12 +3,9 @@ before_install:
   - gem install bundler
   - bundle --version
 rvm:
-  - 2.0.0-p0
-  - 2.0.0-p648
-  - 2.1.0-p0
-  - 2.1.8
-  - 2.2.0-p0
-  - 2.2.4
+  - 2.0.0
+  - 2.1.0
+  - 2.2.0
```

Comment:
This appears to be unrelated to the PR.

The versions were intentional. The idea was to run on the first and the latest release of each minor version.

Reply:
But if you see the Travis logs, it's failing on the `2.0.0-p0` version. I would try to create another PR with those changes. Also would make sense to test for versions that don't have upstream support?

Reply:
Prawn doesn't support 2.0 at this point. I'd suggest this gem maintainers to match Ruby supported version to those of Prawn but ultimately it's up to them.

I was just pointing out that new matrix doesn't cover all versions initially intended. It might be OK if the maintainers decide so.